CHANGES.rst

```diff
@@ -0,0 +1,7 @@
+Changelog
+=========
+
+0.6.2 (unreleased)
+----------------
+
+- Add support for PyPy, Python 2.7, and Python 3.
```

Comment:
I missed the log in `README.txt`, I only noticed `README.tEst`. To keep this PR as simple as possible, I moved it back down there.